packages/core/src/App/Containers/VerificationModal/verification-modal.tsx

```diff
@@ -0,0 +1,112 @@
+import React from 'react';
+import { withRouter } from 'react-router-dom';
+import { DesktopWrapper, MobileDialog, MobileWrapper, Modal, UILoader, AutoHeightWrapper } from '@deriv/components';
+import { ProofOfIdentityContainer } from '@deriv/account';
+import { observer, useStore } from '@deriv/stores';
+import './verification-modal.scss';
+
+type TVerificationModal = {
+    is_from_external?: boolean;
+    onStateChange?: () => void;
+};
+
+const VerificationModal = observer(({ is_from_external, onStateChange }: TVerificationModal) => {
+    const { client, common, notifications, ui } = useStore();
+    const {
+        account_status,
+        account_settings,
+        fetchResidenceList,
+        getChangeableFields,
+        is_switching,
+        is_high_risk,
+        is_withdrawal_lock,
+        should_allow_authentication,
+        is_virtual,
+        updateAccountStatus,
+    } = client;
+    const { refreshNotifications } = notifications;
+    const { app_routing_history, routeBackInApp } = common;
+    const { is_verification_modal_visible, setIsVerificationModalVisible } = ui;
+    return (
+        <React.Suspense fallback={<UILoader />}>
+            <DesktopWrapper>
+                <Modal
+                    className='verification-modal'
+                    is_open={is_verification_modal_visible}
+                    title='Submit your proof of identity and address'
+                    toggleModal={() => setIsVerificationModalVisible(false)}
+                    height='700px'
+                    width='996px'
+                    exit_classname='verification-modal--custom-exit'
+                >
+                    <AutoHeightWrapper default_height={200}>
+                        {({ setRef }) => (
+                            <div ref={setRef} className='proof-of-identity'>
+                                <div className='proof-of-identity__main-container'>
+                                    <ProofOfIdentityContainer
```

Comment:
lets move ProofOfIdentityContainer to a separate component like ModalContent. and reuse for Desktop and mobile version